src/ontology/mondo-edit.obo

```diff
@@ -399910,7 +399910,7 @@ xref: SCTID:238695001 {source="MONDO:equivalentTo"}
 xref: UMLS:C0406409 {source="MEDGEN:kboom-pr98-c99", source="GARD:0010278", source="MONDO:equivalentTo"}
 is_a: MONDO:0005093 {source="linkedlifedata", source="linkedlifedata/inferred"} ! skin disease
 intersection_of: MONDO:0005093 ! skin disease
-intersection_of: disease_has_feature HP:0000989 ! Pruritu
+intersection_of: disease_has_feature HP:0000989 ! Pruritus
```

Comment:
I didn't make this change, it was applied automatically by protege. Presumably because if you edit any part of a term, the entire term is reformatted? @nicolevasilevsky is that the case? This is the same question I have at https://github.com/EBISPOT/efo/issues/885#issuecomment-723233218.

It seems like it'd be ideal for these automated changes to be applied to the entire ontology all at once, rather than upon term edit. This would help keep diffs clean.

Reply:
I think this has to do with the imports being refreshed - it is something weird that Protege does sometimes. In this case, don't worry about it.

Reply:
@matentzn can you please correct me if my explanation is wrong? :)